### PR TITLE
show events on calendar

### PR DESCRIPTION
**Pull request by Karyum**
#104

### CODE REVIEW

File: public/js/helpers/get_startTime.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* global axios Cal addDateEventListeners */
+
+(function () {
+  axios
+    .get('https://nazareth-open-tourism-platform.herokuapp.com/api/v1/events')
+    .then(function (res) {
+      var data = res.data.reduce((acc, event) => {
+        return acc.concat(event.startTime);
+      }, []);
```
Comment (finnhodgkin) [inline public/js/helpers/get_startTime.js]: Why is this not a map?

```js
var data = res.data.map(event => event.startTime);
```

File: public/js/calendar.js [hunk 2]
```diff
@@ -92,10 +113,34 @@ Cal.prototype.showMonth = function (y, m) {
...
+    if (
+      chkY === this.currYear &&
+      chkM === this.currMonth &&
+      i === this.currDay
+    ) {
       html += '<td data-date=' + dstring + ' class="day today">' + i + '</td>';
     } else {
-      html += '<td data-date=' + dstring + ' class="day normal">' + i + '</td>';
+      var currentDate = new Date(this.currYear, this.currMonth, i);
+      for (var h = 0; h < dateArray.length; h++) {
```
Comment (finnhodgkin) [inline public/js/calendar.js]: I know this is someone else's code but can we clean it up a little..? Make this fella a forEach and clear up some of the variable names :pray:

File: public/js/helpers/get_startTime.js (hunk 1 above)
Comment (finnhodgkin) [reply public/js/helpers/get_startTime.js]: Also what is this weird mix of ES6 and ES5? I'm guessing with webpack everything will move to lovely ES6 anyway :+1:

Comment (Karyum) [reply public/js/helpers/get_startTime.js]: why should it be a map when reduce can do everything ? :stuck_out_tongue_winking_eye:

File: public/js/calendar.js (hunk 2 above)
Comment (Karyum) [reply public/js/calendar.js]: cause im using a ` break; ` when i actually find a match with the date

File: public/js/helpers/get_startTime.js (hunk 1 above)
Comment (finnhodgkin) [reply public/js/helpers/get_startTime.js]: While I agree that reduce is the best function, map is close behind and more appropriate here :dancer:

**Pull request merged**